evm/v0.5/test/Chainlink_test.js

```diff
@@ -95,11 +96,11 @@ contract('Chainlink', () => {
     })
 
     it('handles strings', async () => {
-      await cl.addBytes('first', toHex('apple'))
+      await cl.addBytes('first', ethers.utils.toUtf8Bytes('apple'))
       const tx = await cl.closeEvent()
       const [payload] = parseCLEvent(tx)
       const decoded = await decodeDietCBOR(payload)
-      const expected = toBuffer('apple')
+      const expected = ethers.utils.toUtf8Bytes('apple')
       assert.deepEqual(decoded, { first: expected })
     })
   })
```

Comment:
Is this file supposed to be included as part of this PR?

Reply:
Yup, this is a failing test that needed to be fixed, regenerating the lockfile cause non-semver following deps to break some things.

Reply:
Okay! :+1: